app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_mv, file_name, path_new_file = command.split(" ")
```

Comment:
I don't see this:

![image](https://user-images.githubusercontent.com/104218550/201998448-6747aa98-e6cc-47a7-a0cb-f572c7206bd5.png)